docs/backends/Google.md

```diff
@@ -222,6 +222,34 @@ On the Local, SGE, and associated backends any GCS URI will be downloaded locall
 precedence over the `root` specified at `backend.providers.JES.config.root` in the configuration file. Google Cloud Storage URIs are the only acceptable values for `File` inputs for
 workflows using the Google backend.
 
+**Retry with More Memory**
+
+With `memory-retry` you can specify an array of strings which when encountered in the `stderr` file by Cromwell, allows the task to be retried with more memory.
+The optional `multiplier` config specifies the factor by which the memory should be multiplied while retrying. If the value is not mentioned in config, it will default 
+to 2.0. The retry will be counted against the `maxRetries` count mentioned in the `runtimeAtrributes` in the task. For example,
+```hocon
+backend.providers.Papiv2.config {
+  memory-retry {
+    error-keys = ["OutOfMemoryError", "Killed"]
+    multiplier = 1.1
+  }
+}
+```  
+this tells Cromwell to retry the task with 1.1x memory when it sees either `OutOfMemoryError` or `Killed` in the `stderr` file. If the task has 
+runtime attributes as below 
+```hocon
+runtimeAtrributes {
+  memory: "1 GB"
+  continueOnReturnCode: true
+  maxRetries: 1
+}
+``` 
+the task will be retried at max 1 more time, and this time with "1.1 GB" memory. 
```

Comment:
discussed in person that the actual VM specs don't always match exactly what was requested 🙂